test/common-utils/scenarios.json

```diff
@@ -115,6 +115,16 @@
             }
         }
     },
+    "config-subdirectory": {
```

Comment:
Looks like there is a test failure for this scenario, can you help fix it?
https://github.com/devcontainers/features/actions/runs/4910062272/jobs/8781347850?pr=547#step:4:14678

Reply:
Attempted to fix it, but I'm not sure what other directories there would be for me to mount.